types/persisted-model.d.ts

```diff
@@ -519,6 +520,10 @@ export declare class PersistedModel extends ModelBase {
    * @returns {String} The `id` property name
    */
   static getIdName(): string;
+
+  definePersistedModel<T>(entityClass: T): PersistedModel;
```

Comment:
Where is this method implemented? I could not find it in the juggler codebase `grep definePersistedModel -r lib/`.

Reply:
Errata, it's not required.